develop/devguide/Connector/SwarmingScriptElement.md

```diff
@@ -50,6 +50,13 @@ As a first step, create a short Automation script that will swarm a fixed elemen
 
 1. Execute the script to launch a swarming action for the specified element to the specified target host.
 
+   From DataMiner 10.5.9/10.6.0 onwards<!--RN 43196-->, an information event will be generated when an element was successfully swarmed. Example:
+
+   `Swarmed from <DmaName> (<DmaId>) to <DmaName> (<DmaId>) by <UserName>`
+
+   > [!NOTE]
+   > When the source DMA is no longer available or unknown, the information event will be shortened to `Swarmed to <DmaName> (<DmaId>) by <UserName>`.
```

Comment:
Is this a better location to put the info, or would you completely remove it from this page in the development guide, @RobbeDGH?